modules/Hoot/ui/managePanel/transAssist/upload.js

```diff
@@ -3,28 +3,32 @@
  * Project: hootenanny-ui
  * @author Matt Putipong - matt.putipong@radiantsolutions.com on 7/5/18
  *******************************************************************************************************/
+import { d3combobox } from '../../../../lib/hoot/d3.combobox';
+import _find from 'lodash-es/find';
+import _forEach from 'lodash-es/forEach';
+import _map from 'lodash-es/map';
 
 export default class Upload {
     constructor( instance ) {
         this.instance = instance;
-
-        this.schemaOptions = [
-            {
-                name: 'OSM',
-                enabled: true,
-                checked: true
-            },
-            {
-                name: 'TDSv70',
-                enabled: true
-            },
-            {
-                name: 'TDSv61',
+        this.schemaOptions = Hoot.translations.availableTranslations.map(v => {
```

Comment:
as combobox maybe the mapping here is just to a list of objects like `{key: value, value: value}`. I think enabled was a concept tied to when these were radio inputs.